_config.yml

```diff
@@ -0,0 +1,111 @@
+# Site settings
+title: Pyronear
+description: "Détection précoce de feux de forêt" # this means to ignore newlines until "baseurl:"
+baseurl: "" # the subpath of your site, e.g. /blog/
+url: "" # the base hostname & protocol for your site
+
+twitter_username: pyro_near
+facebook_username: pyronear
+linkedin_username: pyronearfr
+github_username: pyronear
+
+# Build settings
+markdown: kramdown
+
+# Traduction
+t:
+  fr:
+    buttons:
+      moreInfo: "En savoir plus"
+      contactUs: "Nous contacter"
+      submit: "Envoyer"
+    header:
+      title: "Détection précoce des départs de feux"
+      subtitle: "Pyronear est un projet open-source au service du bien commun."
+    manifesto:
+      title: "Qui sommes-nous ?"
+      phrase1: "Pyronear est une association loi 1901 au service du bien commun et du logiciel libre."
+      phrase2: "Nous portons la conviction que la protection des forêts doit être accessible à tous et qu'elle nécessite la technologie la plus efficace, transparente et abordable possible."
```

Comment:
Maybe we can add that:
 "Nous portons la conviction que la protection des forêts doit être accessible à tous et qu'elle nécessite la technologie la plus efficace, transparente et abordable possible. C’est pourquoi nous souhaitons proposer gratuitement notre solution pour détecter les feux de forêt. "
In order to make clear that our solution is free

Reply:
Done!